src/main/webapp/3Dscript.js

```diff
@@ -194,8 +330,46 @@ function gui() {
   gui.add(poseTransform, 'scale', .5, 2, .25)
       .onChange(plotOrientation)
       .onFinishChange(plotTrajectory).name('Pose Scale Multiplier');
+
+  const max= pose.length;
+  oldEnd=max;
+  gui.add(poseStart, 'start', 0, max, max/100+1)
+      .onFinishChange(hideOrientationStart);
+  gui.add(poseEnd, 'end', 0, max, max/100+1)
+      .onFinishChange(hideOrientationEnd);
+  /* Time value. */
+  const timeStart = {time: ''};
+  time = gui.add(timeStart, '');
+
+  /* Lat value. */
+  const latStart = {lat: ''};
+  lat = gui.add(latStart, 'lat');
```

Comment:
Can you omit all the `xxStart` variables and make the objects inline?
`lat = gui.add({lat: ''}, 'lat')`